client/src/components/Header.js

```diff
@@ -69,6 +69,13 @@ const Header = () => {
 
 	return (
 		<header>
+		 <div className="background--co1Y8">
+      <img
+        src="https://airtame.com/wp-content/uploads/2019/06/Code-your-future-02-2400x1098.png"
+        className="imageBanner---h3G2"
+        alt="Free mixed hero backgrounds"
```

Comment:
Do you need this alt text? This image is decorative so for partially sighted people using screenreaders there's no benefit to them hearing this text. It can just be ""

https://www.nomensa.com/blog/what-screen-reader/